08-TabbedNavigation/src/PrismSample/Views/TabCPage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:prism="http://prismlibrary.com"
+             prism:ViewModelLocator.AutowireViewModel="True"
+             x:Class="PrismSample.Views.TabCPage"
+             Title="Tab C">
+
+  <StackLayout>
+    <Label Text="Tab C" HorizontalOptions="CenterAndExpand"></Label>
+    <Button Text="Go To Tab A" HorizontalOptions="CenterAndExpand" Command="{Binding TabACommand}"></Button>
+  </StackLayout>
```

Comment:
- Center the StackLayout Vertically
- The label should be using `HorizontalTextAlignment="Center"` not the HorizontalOptions
- All elements like Label / Button that you are not setting complex properties on should be self closing `<Label />`